images/dex/tests/runs.sh

```diff
@@ -4,14 +4,11 @@ set -o errexit -o nounset -o errtrace -o pipefail -x
 
 PORT=5556
 
-URL="http://127.0.0.1:${PORT}"
-docker run --rm -d -p $PORT:$PORT "${IMAGE_NAME}" dex serve /etc/dex/config.docker.yaml
+URL="http://127.0.0.1:${FREE_PORT}"
```

Comment:
We have an asymmetry.  We need to `curl` with `FREE_PORT`, but the issuer output on line `29` should grep the URL with `PORT`.

The current script can use a single `URL` variable because the port are symmetric, but we're changing that 😞